### DISCUSSION TITLE
Per-file individual lint rule ignores

### DISCUSSION THREAD

**Discussion by peniqliotuv** [rome/tools#4673] [Suggestions]
One thing I miss from ESLint is the ability to declare at the top of the file
```
/* eslint-disable no-console */
```
I have a script with a lot of `console.log()`'s, and although I know that I can exclude this specific file from the `rome.json`, I don't want to _**not**_ lint and format it at all. 
It would be nice if we could have something similar, like a `rome-ignore-file <rule> <explanation>`

**Comment by pau1tuck** [comment]
Yes, this would be really helpful. Thanks for raising it @peniqliotuv
